main/guides/zoe/actual-contracts/PSM.md

```diff
@@ -0,0 +1,94 @@
+# PSM Contract
+
+<Zoe-Version/>
+
+##### [View the code on Github](https://github.com/Agoric/agoric-sdk/blob/master/packages/inter-protocol/src/psm/psm.js) (Last updated: Aug 15, 2022)
+
+The Parity Stability Module (PSM) contract allows users to convert Inter Stable Tokens (ISTs) to external stable tokens at a specified fixed ratio, and vice versa.
+
+IST is an evolving, fully collateralized, cryptocurrency-backed decentralized stable token for the interchain ecosystem on the Agoric chain. IST is designed to maintain parity with the US dollar (USD) for broad accessibility and provides Agoric and the interchain ecosystem users an asset with minimum price volatility. In addition, IST is the native fee token for the Agoric platform and provides some of the core functionality and stability for the Agoric cryptoeconomy. Currently the PSM is the only minter of ISTs.
```

Comment:
I find the docs easier to maintain if lines are broken at a reasonable length. It doesn't matter to me where that is between 70 characters and 100, but really long lines make diffs harder to review when we want to make changes later.